docker/Dockerfile

```diff
@@ -22,5 +22,6 @@ RUN install2.r --error \
         socialmixr
 
 RUN Rscript -e 'remotes::install_github(c("mrc-ide/dust",  "mrc-ide/mcstate", "mrc-ide/eigen1", "mrc-ide/odin", "mrc-ide/odin.dust"))'
+RUN Rscript -e 'remotes::install_github(c("mrc-ide/dust@i245-shared-int", "mrc-ide/mcstate@dust-0.9.10"))'
```

Comment:
Codefactor complains about this, but I think it's going away anyway
```suggestion
```

Reply:
yeah, that's an expected complaint, will remove once everything else is merged (having a mare with mcstate....)